CRARS.Rmd

```diff
@@ -135,4 +135,27 @@ Main objectives -
 ## 03/29/2022
 
 Began to develop working protoypes of the intake form to present to the AG representatives.
-Waiting on ITSS approval for Shiney App. 
\ No newline at end of file
+Waiting on ITSS approval for Shiney App. 
+
+# 04/12/2022
+
+## Project Updates
+
+Google Form - 
+
+* Added file download / upload part to the Google form to allow for many samples
+* There is a problem as the sheet is uploaded as a link, could slow down automation
```

Comment:
there is a `download.file()` function in R that can take that URl and download te file into a folder of your choosing